vSLMsTUDY.md

```diff
@@ -1,5 +1,8 @@
 # VLSM slash notes
 
+## Overview ##
+While studying the process of Variable Length Subnet Masking, creating a table for each subnet size helped me to be more efficient at creating different subnets and avoiding the mistakes that were most common for me. This is currently only identifying Class C networks, but may continue into Classes A and B in the future.
```

Comment:
a little unclear as to what the user can do with this. Could reorganize in the following format: 
1. Description of what page is and how it's used. e.g. This page provides tables for class C subnets to help with subnet creation
2. Overview of why the user might care. Similar to starting out. 

Another option is to clarify the page through the title to set a good initial context.

Reply:
I hear that. I will adjust the wording to be a little more useful.